baseplate/experiments/__init__.py

```diff
@@ -77,6 +78,16 @@ def _get_config(self, name):
             logger.warning("Could not load experiment config: %s", str(exc))
             return
 
+    def _get_experiment(self, name):
+        if name not in self._experiment_cache:
```

Comment:
Does something invalidate this if the underlying file changes? This'll cache away use of the filewatcher right?